frameworks/kafka/docs/connecting-clients.md

```diff
@@ -17,74 +17,55 @@ The only supported client library is the official Kafka Java library, i.e., `org
 
 The following command can be executed from the cli in order to retrieve a set of brokers to connect to.
 
-    dcos kafka --name=<name> connection
-    
```

Comment:
dcos kafka endpoints

list all endpoints

there are two

zookeeper and broker